### PR TITLE
LPD-8381  Send user notifications when publication fails to publish

### PR DESCRIPTION
**Pull request by brookedalton**
https://liferay.atlassian.net/browse/LPD-8381

### CODE REVIEW

**Comment by brookedalton** [comment]
ci:test:sf

File: modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/internal/background/task/CTPublishBackgroundTaskExecutor.java [hunk 1]
```diff
@@ -235,6 +244,43 @@ public BackgroundTaskDisplay getBackgroundTaskDisplay(
 		return new CTPublishBackgroundTaskDisplay(backgroundTask);
 	}
 
+	@Override
+	public String handleException(
+		BackgroundTask backgroundTask, Exception exception) {
+
+		if (!FeatureFlagManagerUtil.isEnabled("LPD-11018")) {
+			return StringPool.BLANK;
```
Comment (pei-jung) [inline modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/internal/background/task/CTPublishBackgroundTaskExecutor.java:252]: Return from superclass. `super.handleException`

**Review by pei-jung — APPROVED** [review]

**Comment by pei-jung** [comment]
ci:test:relevant

**Comment by pei-jung** [comment]
ci:test:publications

**Comment by pei-jung** [comment]
See https://github.com/liferay-publications/liferay-portal/pull/987

**Pull request closed without merging**